### PR TITLE
Migrated Performance benchmarking using Meshery and Nighthawk blog po…

### LINKED ISSUE #1069: [NG] Migrate two recent blogs to layer5-ng
**Current Behavior**
Two blogs have been added to layer5 website recently.

**Desired Situation**
Migrate both the blogs to layer5-ng website.

- [ ] [Performance Benchmarking using Meshery and NightHawk
](https://layer5.io/blog/performance-benchmarking-using-meshery-and-nighthawk)
- [ ] [Meshery Adapter for Open Service Mesh
](https://layer5.io/blog/meshery/announcing-the-meshery-adapter-for-open-service-mesh)

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by lolyparty**
…st to layer5-ng website

Signed-off-by: lolyparty <ololadeolayiwola01@gmail.com>

**Description**

This PR fixes #1069 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by akashgp09 — COMMENTED** [review]
Add this [blog](https://layer5.io/blog/meshery/announcing-the-meshery-adapter-for-open-service-mesh) too in the Layer ng website.

**Comment by lolyparty** [comment]
> Add this [blog](https://layer5.io/blog/meshery/announcing-the-meshery-adapter-for-open-service-mesh) too in the Layer5 ng website.

That's what I just did

File: src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+<img src={conformance} className="" alt="meshery-smi-conformance-results-image" />
+
+Try Open Service Mesh now, by [getting started with Meshery](https://github.com/layer5io/layer5/blob/master/meshery#getting-started).
+
+-- [Lee Calcote](https://github.com/layer5io/layer5/blob/master/community/members/lee-calcote)
+
```
Comment (akashgp09) [inline src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx:45]: https://github.com/layer5io/layer5/blob/master/community/members/lee-calcote 
The above link is not working. Change it to (https://layer5ng.netlify.app/community/members/lee-calcote)

**Review by akashgp09 — CHANGES_REQUESTED** [review]

File: src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx (hunk 1 above)
Comment (lolyparty) [reply src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx:45]: > https://github.com/layer5io/layer5/blob/master/community/members/lee-calcote
> The above link is not working. Change it to (https://layer5ng.netlify.app/community/members/lee-calcote)

Oh thanks, I'll do that asap.

File: src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+import conformance from "./conformance-results.png";
+import meshery from "./meshery-open-service.png"
+
+<BlogWrapper>
+
+### Adding another service mesh to the landscape and another adapter to Meshery
```
Comment (Jashpatel1) [inline src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx:20]: This is to be added as a subtitle field in frontmatter.

File: src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Released on August 5th, 2020 by Microsoft, [Open Service Mesh](https://openservicemesh.io/) (OSM) is a lightweight
+
+<img src={meshery} className="image-right" alt="meshery-adapter-for-open-service-image" />
```
Comment (Jashpatel1) [inline src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx:23]: This logo can be removed as the same is already present above title.

**Review by Jashpatel1 — CHANGES_REQUESTED** [review]

File: src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx (hunk 2 above)
Comment (Jashpatel1) [reply src/collections/blog/2020-08-20-announcing-the-meshery-adapter-for-open-service-mesh/index.mdx:20]: @lolyparty as you have already added it to the subtitle, it's no longer required here. It's duplicate. Please remove this.

**Review by Jashpatel1 — APPROVED** [review]
All looks perfect. Thanks for making those changes @lolyparty

**Pull request merged**